src/examples/molecules/Modal/Example.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react'
+import Modal from '../../../components/molecules/Modal'
+import Card from '../../../components/atoms/Card'
+import Button from '../../../components/atoms/Button'
+
+const ModalDefault = () => {
+  const [show, setShow] = useState(false)
+
+  const toggleModal = () => {
+    setShow(!show)
+  }
+  return (
+    <Card style={{ padding: '30px' }}>
+      <Button solid primary onClick={toggleModal}>
+        Open Modal
+      </Button>
+      <Modal
+        showModal={show}
+        modalTitle="modal title"
+        modalContent="Lorem ipsum dolor sit amet."
+        onModalClose={toggleModal}
+        className="modal--relative"
```

Comment:
do we really need this??
even if we need pass the prop and assign the class in components
```
<Modal relative />
<Modal fixed />
```

Reply:
we don't need that @PRASHANTK22111993 I have removed it.

Reply:
@Rajmohanrk do we still need in scss file .

```
  &--relative {
    position: absolute;
  }
```

Reply:
That would not be required as well